src/config.rs

```diff
@@ -149,6 +151,8 @@ impl Config {
     }
 
     pub fn get<'de, T: Deserialize<'de>>(&self, key: &str) -> Result<T> {
+        let k = key.to_lowercase();
+        let key = k.as_str();
```

Comment:
```suggestion
        let key = key.to_lowercase().as_str();
```

Does this work?

Reply:
No I got the  "temporary value dropped while borrowed (rust)" error.  If you know of a more elegant way I will be happy to amend it.